tmail-backend/apps/postgres/src/main/java/com/linagora/tmail/james/app/PostgresTmailServer.java

```diff
@@ -237,7 +260,7 @@ private static Module chooseRspamdModule(PostgresTmailConfiguration configuratio
 
     private static Module chooseJmapModule(PostgresTmailConfiguration configuration) {
         if (configuration.jmapEnabled()) {
-            return new JMAPListenerModule();
+            return Modules.combine(new JMAPEventBusModule(), new JMAPListenerModule());
```

Comment:
`JMAPEventBusModule()` has some instances of Rabbitmq
If we install it here (when jmapEnabled), we will get an error when run with Inmemory eventBus

Reply:
maybe dirty fix: put eventBus choice in this, and if InMemory eventBus then load another dedicated `JMAPEventBusModule` (which does not container RabbitMQ one)?

Reply:
Good catch, makes sense. Will think of a refactoring :)